src/components/common/PromoProductBox/PromoProductBox.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './PromoProductBox.module.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faStar,
+  faExchangeAlt,
+  faShoppingBasket,
+  faEye,
+} from '@fortawesome/free-solid-svg-icons';
+import { faStar as farStar, faHeart } from '@fortawesome/free-regular-svg-icons';
+import Button from '../Button/Button';
+
+const PromoProductBox = ({
+  name,
+  price,
+  stars,
+  photoBackground,
+  oldPrice,
+  isFavourite,
+  compare,
+}) => (
+  <div className={styles.root}>
+    <div
+      className={styles.photo}
+      style={{
+        backgroundImage: `url(${photoBackground})`,
```

Comment:
Zastosuj `img`, powinniśmy wybierać `img` zamiast background image ze wzgędu na SEO, `img` jest indeksowane, background nie ;)

Reply:
Zastosowałem to rozwiązanie ze względu na to że w ten sam sposób robione jest to w komponencie productBox i chciałem utrzymać spójność projektu

Reply:
Ale co zyskujemy dzięki tej spójności kodu? Nawet nie mamy jednego wspólnego komponentu pod to zdjęcie, a tracimy na SEO. Moim zdaniem to raczej ProductBox pasowało by też na `img` przerobić ;)

Reply:
Obrazki przerobione na imgki zamiast tła